tests/unit/charms/tls_certificates_interface/v2/test_tls_certificates_v2_provides.py

```diff
@@ -743,7 +743,7 @@ def test_given_no_certificates_in_relation_data_when_get_issued_certificates_the
         self.create_certificates_relation_with_1_remote_unit()
         self.harness.set_leader(is_leader=True)
         certificates = self.harness.charm.certificates.get_issued_certificates()
-        self.assertEqual(certificates, {})
+        self.assertEqual(certificates, {"tls-certificates-requirer": []})
```

Comment:
please fix test name, as we don't expect an empty dict anymore.
Maybe something along `empty_certificates_list` would be good.

Reply:
Done